packages/core/src/components/list-slider/list-slider.tsx

```diff
@@ -213,13 +225,19 @@ export class AtomListSlider {
     this.handleSwipe()
   }
 
+  handleTouchMove = () => {
+    this.isTouchMoved = true
+  }
+
   handleSwipe = () => {
-    const lastIndex = this.sliderItems.length - 1
-    const isSwipeLeft = this.touchEndX < this.touchStartX
+    if (this.isTouchMoved) {
+      const lastIndex = this.sliderItems.length - 1
+      const isSwipeLeft = this.touchEndX < this.touchStartX
 
-    this.currentIndex = isSwipeLeft
-      ? Math.min(this.currentIndex + 1, lastIndex)
-      : Math.max(this.currentIndex - 1, 0)
+      this.currentIndex = isSwipeLeft
+        ? Math.min(this.currentIndex + 1, lastIndex)
+        : Math.max(this.currentIndex - 1, 0)
+    }
```

Comment:
It is missing unit tests

Reply:
@tassioFront, as aligned with @felipefialho, this component has low test coverage and some false positives that will be addressed at a later time.